Array manipulation - (Hard)

MeenuSowjanya opened the pull request.

Commit:
Added comments - Ruby HR DS Arrays - 2 D Array ( Easy )

Commit:
reverseArray (Easy)

Commit:
leftRotation (Easy)

Commit:
sparseArrays  (Medium)

Commit:
2D Array - Easy

Commit:
Reverse Array - Easy

Commit:
2D_Array

Commit:
Reverse_Array

Commit:
Added meaningful names

Commit:
Added meaningful names

Commit:
leftRotation (Easy)

Commit:
leftRotation (Easy)

Commit:
arrayManipulation (Hard)

arjunmdas commented on Hard/arrayManipulation.rb:
```diff
@@ -0,0 +1,21 @@
+i
+end
+max_value
+end
```
Can you please format and rewrite this for code readability?

MeenuSowjanya replied:
Formatted and rewritten the code for code readabilty

Commit:
Review changes :
1. Added meaningful variable and function names
2. Used shift operators for pushing

Commit:
Review changes 2:
1. Added comments fr code readability
